### PR TITLE
Feat/promotions docs

### PR DESCRIPTION
**Pull request by pj-sp**

### CODE REVIEW

File: docs/fields/index.md [hunk 1]
```diff
@@ -4,4 +4,11 @@
...
 
 <DocCardList />
-```
\ No newline at end of file
+```
+
+In Prisjakt we support the following feed types:
+- Offer feeds
+- Promotion feeds
+- Product feeds
```
Comment (andrejohansson) [inline docs/fields/index.md:12]: This is good, but perhaps we can clarify target audience a bit:

- Offer feeds (used by shops)
- Promotion feeds (used by shops)
- Product feeds (used by manufacturers)

File: docusaurus.config.js [hunk 2]
```diff
@@ -47,6 +47,19 @@ const config = {
...
     },
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        createRedirects(existingPath) {
+          const redirects_path='/fields/offer'
+          // We want to redirect from /docs/<offer_feed_field_name> to /docs/offer/<field_name>
+          if (existingPath.startsWith(redirects_path)) {
+            return existingPath.replace(redirects_path, '/fields');
+          }
```
Comment (andrejohansson) [inline docusaurus.config.js:58]: Isn´t the code above doing the opposite of the comment?

**Comment by andrejohansson** [comment]
General structure change:

image

File: docs/examples/_xml_promotion-min-sample.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+```xml
+<?xml version='1.0' encoding='UTF-8'?>
+<rss xmlns:pj="https://schema.prisjakt.nu/ns/1.0" xmlns:g="http://base.google.com/ns/1.0" version="3.0">
```
Comment (andrejohansson) [inline docs/examples/_xml_promotion-min-sample.mdx:3]: Prisjakt namespace should not be the same as for offers.
We should add a new [namespace definition](https://github.com/Prisjakt/feed-specification/tree/master/static/ns) in this folder and reference it.

File: docs/examples/_xml_promotion-sample.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+```xml
+<?xml version='1.0' encoding='UTF-8'?>
+<rss xmlns:pj="https://schema.prisjakt.nu/ns/1.0" xmlns:g="http://base.google.com/ns/1.0" version="3.0">
```
Comment (andrejohansson) [inline docs/examples/_xml_promotion-sample.mdx:3]: Prisjakt namespace should not be the same as for offers.
We should add a new [namespace definition](https://github.com/Prisjakt/feed-specification/tree/master/static/ns) in this folder and reference it.

File: docs/examples/_xml_promotion-min-sample.mdx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  <channel>
+    <title>Prisjakt Minimal Example Feed</title>
```
Comment (andrejohansson) [inline docs/examples/_xml_promotion-min-sample.mdx:5]: Prisjakt Minimal Promotions Feed

File: docs/examples/_xml_promotion-min-sample.mdx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    <description>This is an example feed with the minimal values required</description>
```
Comment (andrejohansson) [inline docs/examples/_xml_promotion-min-sample.mdx:6]: Minimal values for promotions

File: docs/examples/_xml_promotion-sample.mdx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  <channel>
+    <title>Prisjakt Example Feed</title>
```
Comment (andrejohansson) [inline docs/examples/_xml_promotion-sample.mdx:5]: Prisjakt Example Promotions Feed

File: docs/examples/_xml_promotion-sample.mdx [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    <description>This is an example feed with the minimal required and all accepted values</description>
```
Comment (andrejohansson) [inline docs/examples/_xml_promotion-sample.mdx:6]: promotions feed + remove minimal

File: docs/fields/promotion/coupon_value_type.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+import TabItem from '@theme/TabItem';
+import Anchor from "@site/src/components/anchor"
+import Field from '@site/docs/partials/_field.mdx';
+import ReactMarkdown from 'react-markdown';
+import ChangeLog from '@site/src/components/changelog';
+import RequiredField from '@site/docs/partials/_required_field.md';
+
+# coupon_value_type
+
+<RequiredField/>
```
Comment (andrejohansson) [inline docs/fields/promotion/coupon_value_type.md:15]: This field is not required.

File: docs/fields/promotion/promotion_effective_dates.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+<RequiredField/>
+
+## Description
+
+Defines the time period when the promotion is active.
+
+
+
+
```
Comment (andrejohansson) [inline docs/fields/promotion/promotion_effective_dates.md:23]: We should add a do/dont section or update validation rules to explain logic regarding dates. Such as "you cannot edit a started promotion", "you cannot remove passed promotions" etc. Depends on the business logic in BC.

File: docs/terminology/prisjakt-specific-fields.md [hunk 11]
```diff
@@ -10,8 +10,8 @@ But in order to be able to provide you with extra functionality we sometimes add
 
 Currently, our Prisjakt specific fields are:
 
-- [prisjakt_id](/fields/prisjakt_id.md)
-- [member_price](/fields/member_price.md)
+- [prisjakt_id](/fields/offer/prisjakt_id.md)
+- [member_price](/fields/offer/member_price.md)
```
Comment (andrejohansson) [inline docs/terminology/prisjakt-specific-fields.md:15]: We should have three sub headings here "offers, promotions, products" and explain what fields we have extended with.

**Review by andrejohansson — CHANGES_REQUESTED** [review]
Nice work, some things to adjust then we are good to go!

File: docs/fields/index.md (hunk 1 above)
Comment (pj-sp) [reply docs/fields/index.md]: done

File: docusaurus.config.js (hunk 2 above)
Comment (pj-sp) [reply docusaurus.config.js:70]: added a description in a comment

File: docs/fields/promotion/promotion_effective_dates.md (hunk 10 above)
Comment (pj-sp) [reply docs/fields/promotion/promotion_effective_dates.md:23]: added

File: docs/terminology/prisjakt-specific-fields.md (hunk 11 above)
Comment (pj-sp) [reply docs/terminology/prisjakt-specific-fields.md:16]: done

File: docs/examples/_xml_promotion-sample.mdx (hunk 4 above)
Comment (pj-sp) [reply docs/examples/_xml_promotion-sample.mdx]: done

File: docs/examples/_xml_promotion-min-sample.mdx (hunk 5 above)
Comment (pj-sp) [reply docs/examples/_xml_promotion-min-sample.mdx]: done

File: docs/examples/_xml_promotion-sample.mdx (hunk 8 above)
Comment (pj-sp) [reply docs/examples/_xml_promotion-sample.mdx]: done

**Review by andrejohansson — APPROVED** [review]

**Pull request merged**